packages/components/src/lib/SongCard.tsx

```diff
@@ -38,10 +53,33 @@ const SongCard = ({
   priceInNewm,
   priceInUsd,
   subtitle,
-  isLoading = false,
+  title,
 }: SongCardProps) => {
   const theme = useTheme();
 
+  const [songProgress, setSongProgress] = useState(0);
+
+  useEffect(() => {
+    if (isPlaying && duration) {
+      // Gives indication that the song is playing by incrementing the progress bar
+      if (songProgress === 0) setSongProgress(1);
```

Comment:
I added this to show progress of the playback from the start of track audio, but no issue removing it too. I think we need to add an extra one second to the duration because of this, but would like to see your and product's opinion.

Standard across other platforms: Would need to remove the line, the first second of progress is typically delayed by the interval loop. Seen on Spotify and others.